source/whats-new.txt

```diff
@@ -36,43 +36,122 @@ Upcoming Breaking Changes
+   To avoid breaking changes in future major releases of the driver,
+   replace any application code that depends on the deprecated code.
+
+The 4.11 driver release deprecates the following items:
+
 - The ``getStats()`` and ``isCapped()`` instance methods of the
-  ``DBCollection`` class are deprecated in v4.11. The corresponding server
-  commands are deprecated in MongoDB v6.2 and later. Use the ``$collStats`` aggregation
-  pipeline stage to retrieve the information provided by these methods
-  instead. You can run the aggregation as shown in the following code example:
+  ``DBCollection`` class are deprecated. The corresponding server
+  commands are deprecated in MongoDB v6.2 and later. Use the ``$collStats``
+  aggregation pipeline stage to retrieve the information provided by these
+  methods instead. You can run the aggregation as shown in the following code
+  example:
 
   .. code-block:: java
 
-     collection.aggregate(Arrays.asList(
-                      new Document("$collStats",
-                          new Document("storageStats", new Document()))));
+     Cursor cursor = collection.aggregate(Arrays.asList(
+             new BasicDBObject("$collStats",
+                     new BasicDBObject("storageStats", new BasicDBObject()))),
+             AggregationOptions.builder().build()
+     );
 
-  To determine whether the collection is a capped collection, access the value
-  of the ``storageStats.capped`` field returned by this example aggregation.
+  To determine whether a collection is a capped collection, access the value
+  of the ``storageStats.capped`` field returned by ``Cursor`` instance in the
+  preceding example aggregation.
 
   To learn more about the ``$collStats`` aggregation operator, see the
   :manual:`$collStats (aggregation) </reference/operator/aggregation/collStats/>`
   Server manual entry.
 
-- `MapReduceIterable <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MapReduceIterable.html>`__
-  and map-reduce methods on `MongoCollection <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#mapReduce(com.mongodb.client.ClientSession,java.lang.String,java.lang.String)>`__
-  are deprecated in MongoDB v4.2 and later. They will be replaced by the
-  aggregation framework and removed in a future release.
-- Beginning with v5.0, the {+driver-short+} will require Java 11 or later.
+- The following network address-related methods are deprecated: (TODO: replace this item with https://github.com/mongodb/docs-java/pull/446/files)
 
-.. _version-4.11:
+  - ``getSocketAddress()`` and ``getSocketAddresses()`` methods from the `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__ class
+  - ``getUnixSocketAddress()`` from the `UnixServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/UnixServerAddress.html>`__ class
 
-What's New in 4.11
-------------------
+- The following Stream API methods and classes are deprecated and are
```

Comment:
The following Stream API. -> The following Stream-related API.

Also, can Stream be a link to the Stream interface, or at least highlighted as a type?